src/SDKs/ProvisioningServices/Management.ProvisioningServices/Microsoft.Azure.Management.ProvisioningServices.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.ProvisioningServices</PackageId>
+    <Description>Provides management capabilities for Microsoft Azure IoT Device Provisioning Service.</Description>
+    <AssemblyTitle>Microsoft Azure Provisioning Service Management</AssemblyTitle>
+    <AssemblyName>Microsoft.Azure.Management.ProvisioningServices</AssemblyName>
+    <VersionPrefix>1.0.0</VersionPrefix>
```

Comment:
@montgomp if this is your first package, please release your first preview package as 0.9.0-preview
Once you are ready to go stable, release it as 1.0.0 (without preview)
make similar changes in AssemblyInfo.cs